apps/src/StudioApp.js

```diff
@@ -43,6 +43,12 @@ var ENGLISH_LOCALE = 'en_us';
  */
 var MAX_PHONE_WIDTH = 500;
 
+/**
+ * Object representing everything in window.appOptions (often passed around as
+ * config)
+ * @typedef {Object} AppOptionsConfig
+ */
```

Comment:
Haha. I kind of like the idea - especially since the name gives a little more clarity that this is the same thing as appOptions.